rust-runtime/aws-smithy-runtime/src/client/defaults.rs

```diff
@@ -164,6 +166,59 @@ pub fn default_identity_cache_plugin() -> Option<SharedRuntimePlugin> {
     )
 }
 
+/// Runtime plugin that sets the default stalled stream protection config.
+///
+/// By default, when throughput falls below 1/Bs for more than 5 seconds, the
+/// stream is cancelled.
+pub fn default_stalled_stream_protection_config_plugin() -> Option<SharedRuntimePlugin> {
+    Some(
+        default_plugin(
+            "default_stalled_stream_protection_config_plugin",
+            |components| {
+                components.with_config_validator(SharedConfigValidator::base_client_config_fn(
+                    validate_stalled_stream_protection_config,
+                ))
+            },
+        )
+        .with_config(layer("default_stalled_stream_protection_config", |layer| {
+            layer.store_put(
+                StalledStreamProtectionConfig::new_enabled()
+                    .grace_period(Duration::from_secs(5))
```

Comment:
this is the default anyway—I'm a little worried about the default coming from two places because someone might think they're changing it but it won't actually change anything.

maybe the answer is a comment next to the default in `options.rs`?